java/codebuild/reportgroup/README.md

```diff
@@ -0,0 +1,18 @@
+# Welcome to your CDK Java project!
+
+It is a Maven-based project, so you can open this directory with any Maven-compatible Java IDE, and you should be able to build and run tests from your IDE.
+
+You should explore the contents of this template. It demonstrates a CDK app to create a codebuild ReportGroup cloud formation resource.
```

Comment:
What is a ReportGroup? Why is it important to know?
This is a good place for a link to a description, or for a reasonable rundown of what ReportGroups are.

Reply:
Updated